src/wire/igmp.rs

```diff
@@ -31,7 +31,7 @@ mod field {
     use wire::field::*;
 
     pub const TYPE: usize = 0;
-    pub const MAX_RESP_TIME: usize = 1;
+    pub const MAX_RESP_CODE: usize = 1;
```

Comment:
The field is called Max Resp Code in the RFC, and it's not (always) in centiseconds.